src/index.js

```diff
@@ -48,4 +56,8 @@ export const connect = (...args) => {
       return h(Comp, Object.assign({}, props, state, this.actionCreators))
     }
   }
+
+  return (props) => h(StoreContext.Consumer, undefined, (store) => {
```

Comment:
The only way to consume context in the new API is through the context `Consumer` or the [`useContext` hook](https://preactjs.com/guide/v10/hooks#usecontext). This seems like the simplest solution.

Reply:
I like this, very clean.